### DISCUSSION TITLE
Several questions about data sharing

### DISCUSSION THREAD

**Discussion by bc38** [jvde-github/AIS-catcher#300] [Q&A] [answered]
1. I have an AIS server that aggregates AIS data from several base stations and sends it to the Network. Can I submit this data via TCP to Ais-Catcher?
2. Is it possible to run Ais-Catcher without a Dongle, but only with the input data from question #1?
3. Is there a direct IP:port that can be used to transmit AIVDM data to the aiscatcher.org community without running Ais-Catcher?

**Comment by jvde-github** [comment, marked as answer]
1. You can if your Server can enable a TCP server where others can connect to. Let's assume this server runs at IP 192.168.1.2 port 4100, then run AIS-catcher -t txt 192.168.1.2 4100 ....
2. Yes, first bullet as an example. You can also run AIS-catcher to read data as a UDP server/listener. E,g, if AIS-catcher runs on 192.168.1.5, then you can send raw NMEA over UDP to this address and port 4101 and receive it in AIS-catcher as AIS-catcher -x 192.168.1.5 4101
3. Yes, just open a port by filling in this form: https://aiscatcher.org/addstation_udp. I will then create a port where you can send your data as raw NMEA (i.e. AIVDM) over UDP

Unfortunately, for the last item, it needs to be UDP (as it is a bit of an industry standard). You can fill in bogus information in the field if you want to stay anonymous. I only need to have an email address to send you the port number. 

Does that help?

**Comment by bc38** [comment]
Thank you! I will delve into it and figure it out
I have already connected the station via UDP
